### ISSUE TITLE
Extension Development: Failed to load resource: net::ERR_UNKNOWN_URL_SCHEME

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#98231]
- VSCode Version: 1.45.1
- OS Version: Windows 10

# Context
Load a resource like so:
```ts
const vsuri = webview.asWebviewUri(
  Uri.file(
    path.join(__dirname, "../drawio/src/main/webapp/index.html")
  )
);
webview.html = `...  ${vsuri.toString()} ...`; // this loads something from vsuri
```

Launch the Extension Development Host VS Code Instance and ensure that it loads this webview on startup.
You can do this by registering a custom editor that uses such a webview and open such a custom editor before closing the Extension Development Host. The next time the Extension Development Host is started, the custom editor will load the webview. This usually (>70% chance) produces the following error:

# The Error
![image](https://user-images.githubusercontent.com/2931520/82431409-e2d70f80-9a8e-11ea-8317-1f45e970146a.png)

Failed to load resource: net::ERR_UNKNOWN_URL_SCHEME

I guess it refers to `vscode-resource://`.

It only happens on startup, and possibly only in the extension dev host. If I close the the webview and reopen it, it works (this rules out a bug in my extension).

A minimal reproducible extension is some serious work. [Others](https://github.com/hediet/vscode-drawio/pull/45#issuecomment-631363644) have this same issue.

**Comment by hediet** [comment]
It actually also happens when bundling the extension and installing it manually, but much less often (like once for 10 refreshes). So it is not a development only issue.

**Comment by hediet** [comment]
It already happens for the markdown preview feature:

![image](https://user-images.githubusercontent.com/2931520/82434058-a0afcd00-9a92-11ea-816d-44ee946f0d35.png)

You can see that it did not load the style sheet:
![image](https://user-images.githubusercontent.com/2931520/82434110-b32a0680-9a92-11ea-814f-379b75c8a6d7.png)

I haven't managed to reproduce it with all extension disabled though. I guess this bug is timing related.

**Comment by Speedy37** [comment]
Might be fixed by #97777

**Comment by hediet** [comment]
Ah. Duplicate of #89038.
